components/table/interface.tsx

```diff
@@ -83,10 +83,16 @@ export interface FilterDropdownProps {
   prefixCls: string;
   setSelectedKeys: (selectedKeys: React.Key[]) => void;
   selectedKeys: React.Key[];
+  /**
+   * Confirm filter value, if you want to close dropdown before commit, you can call with
+   * {closeDropdown: true}
+   */
   confirm: (param?: FilterConfirmProps) => void;
   clearFilters?: () => void;
   filters?: ColumnFilterItem[];
   visible: boolean;
+  /** Only close filterDropdown */
+  close: () => void;
```

Comment:
Is `visible` a better solution to control visibililty of filterDropdown?

Reply:
if use `visible` to control visibility, we should use a state to manage this state outside the filterDropdown。But this state usually does nothing else。So I think using the 'close' method might be a better choice